client/index.jsx

```diff
@@ -82,6 +82,16 @@ class App extends React.Component {
     });
   }
 
+  handleKeyPress(e) {
+    if (e.key === 'ArrowLeft') {
+      this.previousSlide();
+    } else if (e.key === 'ArrowRight') {
+      this.nextSlide();
+    } else if (e.key === 'Escape') {
+      this.closeCarousel();
+    }
```

Comment:
This might be obvious, but I really like how clearly spelled out your function names are.